src/styles/_catalog.scss

```
@@ -1,16 +1,23 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, $heightCard);
```

You don't need variable $heightCard here, you should use variable $widthCard.
```suggestion
  grid-template-columns: $widthCard;
```

---

fixed